validator/project_validator.go

```diff
@@ -2280,6 +2280,15 @@ func checkTasks(project *model.Project) ValidationErrors {
 				},
 			)
 		}
+		if task.Priority > model.MaxConfigSetPriority {
```

Comment:
Sorry if I wasn't that clear - this only checks the task definition's priority setting. We would also need to check the priority on each task in the build variant task list since that can override the task definition's priority.